Style and other improvements for get_started and ros_installation (#343)
gazebosim/docs: get_started.md

| Mac Monterey | [Gazebo Garden](/docs/garden/install_osx) (recommended), [Gazebo Fortress](/docs/fortress/install_osx) and [Gazebo Citadel](/docs/citadel/install_osx)
| Mac BigSur | [Gazebo Garden](/docs/garden/install_osx) (recommended), [Gazebo Fortress](/docs/fortress/install_osx) and [Gazebo Citadel](/docs/citadel/install_osx)
| Mac Catalina | [Gazebo Garden](/docs/garden/install_osx) (recommended), [Gazebo Fortress](/docs/fortress/install_osx) and [Gazebo Citadel](/docs/citadel/install_osx)

> | Windows | Support via conda-forge is not fully functional, there are known runtime problems [Gazebo for Windows Issue](https://github.com/gazebosim/gz-sim/issues/168).

Reviewer: ```suggestion
| Windows | Support via Conda-Forge is not fully functional, as there are known runtime issues [see this issue for details](https://github.com/gazebosim/gz-sim/issues/168).
```
